easybuild/easyconfigs/a/arrow/arrow-6.0.0.2-foss-2021a-R-4.1.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'RPackage'
+
+name = 'arrow'
```

Comment:
This will clash with `Arrow` on a lowercase module naming scheme. Possibly use `arrow-R`?

We should also rename https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/a/arrow/arrow-0.17.1-foss-2020a-R-4.0.0.eb

Reply:
I don't consider that a problem due to the `versionsuffix` we're using?

If it would be a totally different "arrow", sure, but in this case it actually makes sense to have them under the same "flag"?

Reply:
But, in a lowercase module naming scheme, both modules will be named `arrow` and contain `conflict("arrow")`. And, so, not be loadable at the same time. I don't think a versionsuffix solves that issue.

Though I'd not realised that we don't yet have such a scheme. See https://github.com/easybuilders/easybuild-framework/pull/3199 for a proposed `LowercaseEasyBuildMNS` where I expect this to be problematic.

Reply:
Oh right, I totally overlooked that, very good point...

That's definitely enough reason to rename to `arrow-R` (for lack of a better obvious name)

Reply:
fixed in e6a3f2b + 67cc17f

existing `arrow` easyconfig renamed in #14262